plugin/web/extensions/sidebar/popups/mobileDropdown/index.tsx

```diff
@@ -67,9 +81,17 @@ const MobileDropdown: React.FC = () => {
 
   const handleSearch = useCallback(({ target: { value } }: React.ChangeEvent<HTMLInputElement>) => {
     setSearchTerm(value);
-    postMsg({ action: "saveSearchTerm", payload: { searchTerm: value } });
+    postMsg({ action: "saveSearchTerm", payload: { searchTerm: value, dataSource: "plateau" } });
   }, []);
 
+  const handleCustomSearch = useCallback(
+    ({ target: { value } }: React.ChangeEvent<HTMLInputElement>) => {
+      setCustomSearchTerm(value);
+      postMsg({ action: "saveSearchTerm", payload: { searchTerm: value, dataSource: "custom" } });
+    },
+    [],
+  );
```

Comment:
Maybe not needed (just pass datasource in to callback)

Reply:
Updated base on talk from slack. Now using single searchTerm and it will be cleared when switch tabs.